### DISCUSSION TITLE
Emit subscriber events on pivot tables if entities exist

### DISCUSSION THREAD

**Discussion by jeroni** [mikro-orm/mikro-orm#1974] [Ideas]
Shouldn't be possible to emit events to pivot table entities if entities exists?

I'm listening to changes to entities to emit a socket push event to client side and I expected that if the pivot table entity existed its events would be fired.

**Comment by B4nan** [comment]
You can use flush events and get the collection updates from the UoW instance. See https://github.com/mikro-orm/mikro-orm/issues/788

**Reply by jeroni** [reply]
Thank you very much!
